swagger_generated.json

```diff
@@ -7927,12 +7927,15 @@
         "hexstring" : {
           "type" : "string"
         },
-        "fundingaddr" : {
-          "type" : "string"
+        "addresses" : {
+          "type" : "array",
+          "items" : {
+            "type" : "string"
+          }
         }
       },
       "example" : {
-        "fundingaddr" : "fundingaddr",
+        "addresses" : [ "addresses", "addresses" ],
```

Comment:
Change example to ["list-of-addresses", "last-address-in-list-is-used-for-funding"]